content/en/docs/concepts/deployment-models/index.md

```diff
@@ -0,0 +1,421 @@
+
+### Multiple clusters
+
+You can configure your mesh to include multiple clusters; such configurations
+are known as **multi-cluster**. Using multiple clusters
+affords the following capabilities beyond that of a single cluster model:
+
+- Fault isolation and failover: `cluster-1` goes down, failover to `cluster-2`.
+- Location-aware routing and failover: Send requests to the nearest service.
+- Configuration isolation: A wrong configuration only affects a single cluster.
+- Various [control plane models](#control-plane-models): Support different
+  levels of availability.
+- Team or project isolation: Each team runs its own set of clusters.
+
+{{< image width="50%"
+    link="./multi-cluster.svg"
+    alt="A service mesh with multiple clusters"
+    title="Multi-cluster"
+    caption="A service mesh with multiple clusters"
+    >}}
+
+Multi-cluster deployments give you a greater degree of isolation and
+availability but increase complexity. If your systems have high availability
+requirements, you likely need clusters across multiple zones and regions. You
+can canary configuration changes or new binary releases in a single cluster,
+where the configuration changes only affect a small amount of user traffic.
+Additionally, if a cluster has a problem, you can temporarily route traffic to
+nearby clusters until you address the issue.
+
+You can configure inter-cluster communication based on the
+[network](#network-models) and the options supported by your cloud provider. For
+example, if two clusters reside on the same underlying network, you can enable
+cross-cluster communication by simply configuring firewall rules.
+
+## Network models
+
+Many production systems require multiple networks or subnets for isolation
+and high availability. Istio supports spanning a service mesh over a variety of
+network topologies.
+
+### Single network
+
+In the simplest case, a service mesh operates over a single fully
+connected network. A single network model satisfies the following conditions:
+
+- The network consists of a number of connected subnets without any
+  overlap in IP or VIP ranges.
+- All {{< gloss >}}workload instance{{< /gloss >}}s can reach each other directly
+  without an Istio gateway.
+
+A single network allows Istio to configure service consumers in a uniform
+way across the mesh with the ability to directly address workload instances.
+
+{{< image width="50%"
+    link="./single-net.svg"
+    alt="A service mesh with a single network"
+    title="Single network"
+    caption="A service mesh with a single network"
+    >}}
+
+### Multiple networks
+
+You can span a single service mesh across multiple networks; such a
+configuration is known as **multi-network**.
+
+Multiple networks afford the following capabilities beyond that of single networks:
+
+- An existing network topology
+- Overlapping IP or VIP ranges for **service endpoints**
+- Crossing of administrative boundaries
+- Fault tolerance
+- Scaling of network addresses
```

Comment:
Similarly to my comment about subnets and IP ranges - the list above is interesting to networking people or cloud providers or infrastructure providers. As a user of multiple clusters, why should I be interested in this list? I just order my clusters from my cloud provider(s) or my infrastructure team, and they setup the networks for the clusters for me. Why should a typical Istio user bother themselves with the details above?

Reply:
I think the list is needed. We need to define why you would use one model vs the other, and this is why. Yes, perhaps it a bit into details, but that's also where the differentiation lies. I don't think this list is pulling down the abstract nature of this document much, if any ... it's just giving a brief introduction into why multiple networks is a thing.

Reply:
The question who is the audience. Will the readers setup their networks or will they receive clusters from their cloud providers/infrastructure teams with given networking. My guess is that many readers will not understand these networking concepts.

Reply:
I don't think we can make assumptions either way. We can't make it a gate that every possible reader of this document will understand every word. If you're in a networking section of a document, I think it's fair that you'll see some terminology related to networking. The level of depth here is not a significant hurdle, and there is nothing preventing the user from just reading on.

Reply:
Should we add "some compliance standards require network segmentation"?

From PCI DSS:

>Network segmentation of, or isolating (segmenting), the cardholder data environment from the remainder of an entity’s network is not a PCI DSS requirement. However, it is strongly recommended as a method that may reduce: 
•The scope of the PCI DSS assessment
•The cost of the PCI DSS assessment
•The cost and difficulty of implementing and maintaining PCI DSS controls
•The risk to an organization (reduced by consolidating cardholder data into fewer, more controlled locations) 
Without adequate network segmentation (sometimes called a "flat network") the entire network is in scope of the PCI DSS assessment. Network segmentation can be achieved through a number of physical or logical means, such as properly configured internal network firewalls, routers with strong access control lists, or other technologies that restrict access to a particular segment of a network. To be considered out of scope for PCI DSS, a system component must be properly isolated (segmented) from the CDE, such that even if the out-of-scope system component was compromised it could not impact the security of the CDE.